src/shape.rs

```diff
@@ -126,19 +144,34 @@ pub trait Shape: Sized {
     /// and -1 when it is inside a negative area shape. Of course, greater
     /// magnitude values are also possible when the shape is more complex.
     ///
+    /// The tolerance parameter behaves the same as in [`path_elements()`],
+    /// allowing the default implmentation that first converts to a bezier path.
+    /// As a result, the returned value is allowed to be the tinging number of
```

Comment:
winding number. I think also put `tolerance` in backticks to make it clear we're referring to the parameter.